lib/liveman/survey/surveys.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Liveman.Survey.Surveys do
+  use Ecto.Schema
+
+  alias Liveman.Survey.Schemas.Question
+  alias Liveman.Survey.Schemas.Survey
+
+  @json_file "priv/repo/data/surveys.json"
+
+  def list_surveys do
+    survey_list_json = get_json()
+
+    Enum.map(survey_list_json, fn survey_json ->
+      build_survey(survey_json)
+    end)
+  end
+
+  def get_json do
```

Comment:
In terms of naming, this method is not very descriptive. How about instead:

```suggestion
  def fecth_surveys_from_file do
```